ruby/vampires.rb

```diff
@@ -0,0 +1,83 @@
+insurance = ''
+year_of_birth = ''
+age = ''
+age_calc = 0
+age_true = false
+age_false = false
+insurance_yes = false
+insurance_no = false
+like_garlic = false
+hate_garlic = false
+
+puts "How many candidates are there?"
+candidate_number = gets.chomp
+index = 0
+
+while index < candidate_number.to_i
+
+	puts "What is your name?"
+	employee_name = gets.chomp
+
+	puts "How old are you?"
+	age = gets.chomp.to_i
+
+	puts "What year were you born?"
+	year_of_birth = gets.chomp.to_i
+	age_calc = 2017 - year_of_birth
+
+	if age == age_calc
```

Comment:
LIke what you are doing here, but here is a shorter version; we will leverage the fact `age == age_calc` returning a boolean value

correct_age = age == age_calc

Additionally, if you need to see if someone got an incorrect age you would prepend correct_age with a bang like so, !correct_age